### PR TITLE
add deployment and operation FAQ

### PR DESCRIPTION
**Pull request by glkappe**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: faq/deploy-and-maintain-faq.md [hunk 1]
```diff
@@ -0,0 +1,522 @@
+---
+title: 部署运维 FAQ
+summary: 介绍 TiDB 集群运维部署的常见问题、原因及解决方法。
+category: FAQ
+---
+
+# 部署运维 FAQ
+
+## 一、安装部署
```
Comment (TomShawn) [inline faq/deploy-and-maintain-faq.md:9]: 不建议加序号，建议保留原来格式。加了编号后，之后插入内容还得全局调整。请撤掉该 PR 包含的序号。

File: faq/deploy-and-maintain-faq.md [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+
+TiDB 作为一款开源分布式 NewSQL 数据库，可以很好的部署和运行在 Intel 架构服务器环境及主流虚拟化环境，并支持绝大多数的主流硬件网络，作为一款高性能数据库系统，TiDB 支持主流的 Linux 操作系统环境，具体可以参考 TiDB 的[官方部署要求](/hardware-and-software-requirements.md)。
+
+其中 TiDB 在 CentOS 7.3 的环境下进行大量的测试，同时也有很多这个操作系统的部署最佳实践，因此，我们推荐客户在部署 TiDB 的时候使用 CentOS 7.3+ 以上的Linux 操作系统。
+
+## 硬件要求 FAQ
+
+TiDB 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器平台。对于开发、测试、及生产环境的服务器硬件配置有以下要求和建议：
+
+**开发及测试环境**
```
Comment (zz-jason) [inline faq/deploy-and-maintain-faq.md:31]:
```suggestion
### 开发及测试环境
```

File: faq/deploy-and-maintain-faq.md [hunk 3]
```diff
@@ -0,0 +1,517 @@
...
+
+| **组件** | **CPU** | **内存** | **本地存储** | **网络** | **实例数量(最低要求)** |
+| --- | --- | --- | --- | --- | --- |
+| TiDB | 8核+ | 16 GB+ | SAS, 200 GB+ | 千兆网卡 | 1（可与 PD 同机器） |
+| PD | 8核+ | 16 GB+ | SAS, 200 GB+ | 千兆网卡 | 1（可与 TiDB 同机器） |
+| TiKV | 8核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 3 |
+|   |   |   |   | 服务器总计 | 4 |
+
+**线上环境**
```
Comment (zz-jason) [inline faq/deploy-and-maintain-faq.md:40]:
```suggestion
### 线上环境
```

**Comment by yikeke** [comment]
PTAL again @zz-jason

**Pull request merged**